loops.js

```diff
@@ -50,4 +68,14 @@
 // >> 8
 // >> 9
 // ==========================================
-
+console.log("======== Bonus ===========")
+console.log("in de opdracht staat 'tot 9 logt', echter in het vb is 't/m 9' gelogd, ik heb het vb aangehouden.")
+for (let i = 0; i <= 9; i++){
```

Comment:
Novi geeft een iets andere uitwerking - kan beide volgens mij:

for (let i = 0; i < 10; i++) {
  if (i < 3) {
    console.log(i);
  }
  if (i >= 3 && i < 6) {
    console.log('> ' + i);
  }
  if (i >= 6) {
    console.log('>> ' + i);
  }
}